databricks_blueprints/helpers.py

```diff
@@ -30,38 +32,47 @@ def get_headers(self):
     def get(self, endpoint, params={}):
         api_headers = self.get_headers()
         endpoint_url = self.base_url + endpoint
-        response = requests.get(endpoint_url,
-                                headers=api_headers,
-                                params=params)
-        if response.status_code == 401:  # invalid account token
-            sys.exit(errors.EXIT_CODE_INVALID_CREDENTIALS)
-        elif response.status_code == 404:  # wrong instance_id
-            sys.exit(errors.EXIT_CODE_INVALID_INSTANCE)
+        try:
+            response = self.request.get(endpoint_url,
+                                    headers=api_headers,
+                                    params=params)
+            if response.status_code == 403:  # invalid account token
+                print(f"Invalid Access Token: {self.token}")
+                sys.exit(errors.EXIT_CODE_INVALID_CREDENTIALS)
+        except Exception as e:
+            print(f"Error sending request to {endpoint}: {e}")
+            sys.exit(errors.EXIT_CODE_UNKNOWN_ERROR)
         return response
 
     def post(self, endpoint, data={}):
         api_headers = self.get_headers()
         endpoint_url = self.base_url + endpoint
-        response = requests.post(endpoint_url,
-                                 headers=api_headers,
-                                 json=data)
-        if response.status_code == 401:  # invalid account token
-            sys.exit(errors.EXIT_CODE_INVALID_CREDENTIALS)
-        elif response.status_code == 404:  # wrong instance_id
-            sys.exit(errors.EXIT_CODE_INVALID_INSTANCE)
+        try:
+            response = self.request.post(endpoint_url,
+                                     headers=api_headers,
+                                     json=data)
+            if response.status_code == 403:  # invalid account token
+                print(f"Invalid Access Token: {self.token}")
+                sys.exit(errors.EXIT_CODE_INVALID_CREDENTIALS)
+        except Exception as e:
+            print(f"Error sending request to {endpoint}: {e}")
+            sys.exit(errors.EXIT_CODE_UNKNOWN_ERROR)
         return response
 
     def stream(self, endpoint, json={}):
         api_headers = self.get_headers()
         endpoint_url = self.base_url + endpoint
-        response = requests.post(endpoint_url,
-                                 headers=api_headers,
-                                 json=json,
-                                 stream=True)
-        if response.status_code == 401:  # invalid account token
+        try:
+            response = self.request.post(endpoint_url,
+                                     headers=api_headers,
+                                     json=json,
+                                     stream=True)
```

Comment:
Let's factor out a `check_response` or the like that will check response status in one place instead of copying logic. Specifically looking to eliminate the status_code 403 check and exit code in multiple places.